doc/sections/ex-gwe-vsc.tex

```diff
@@ -0,0 +1,39 @@
+
+% a figure
+\begin{StandardFigure}{
+    The /mf simulation setup includes two GWF models (blue rectangles), two GWT models (green rectangles), and one GWE model (orange rectangle). Solid arrows highlight the exchange of information among the models included in the simulation.  Dashed arrows show the generation of output used to highlight the effects of viscosity on a solute transport model.}
+    {fig:vsc-test-setup}{../images/vsc-test-setup.png}
+\end{StandardFigure}
+
+The initial temperature of the GWE model is 4$^{\circ}C$ which also happens to be the temperature of the groundwater entering the model domain through the constant head boundary on the left side of the model.  Inputs to the VSC package, including parameters for the nonlinear viscosity formulation, are listed in table ~\ref{tab:ex-gwe-vsc-01}.  Within /mf, specifically the VSC package (within the GWF model), pointers to the simulated groundwater temperature enable the VSC package to modify intercell conductance calculated by the node-property flow (NPF) package.  Altered intercell conductance in turn affects the flow solution used by the duplicated GWT model (fig.~\ref{fig:vsc-test-setup}) that would otherwise be the same as the original GWT model ultimately resulting in an altered transport solution, leading to the comparison shown next.
+
+% add static parameter table(s)
+\input{../tables/ex-gwe-vsc-01}
+
+% brief description of results
+\subsection{Example Results}
+
+Output from each GWT model is collected and displayed in figure ~\ref{fig:ex-gwe-vsc-conc-2plts}. Using simple visual inspection differences are apparent between the two plots; however, the magnitude of the difference is more readily observed by differencing the plots, as shown in Figure~\ref{fig:ex-gwe-vsc-diff-02}.  Wherever this difference is negative, meaning the concentrations in the viscosity model run are greater than the no-viscosity model run, the calculated concentrations in the no viscosity model are considered under predicted.  Conversely, higher concentrations in the no viscosity model run compared to the with viscosity model suggest concentrations are over predicted when neglecting viscosity.
+
+It is vital to emphasize the contrived nature of this example.  Given that the calculated temperature field is constant throughout the model domain, the so-called ``effects'' of viscosity shown in Figure~\ref{fig:ex-gwe-vsc-diff-02} would normally be  swamped by the uncertainty of the hydraulic conductivity field.  A more appropriate application of the viscosity package is likely in settings where stark contrasts in groundwater temperature have been observed that may affect the flow field.  In such a setting, activation of the viscosity package may appropriately adjust the intercell conductance values in response to viscosity and reduce compansatory adjustment of the hydraulic conductivity field during a calibration routine, for example.
```

Comment:
Maybe I'm not understanding this example, but it seems like the use of GWE here is not really necessary since it results in a constant temperature field.  For this problem, the effects of viscosity could be evaluated by simply adjusting the hydraulic conductivity value entered to the GWF-NPF package.  Or maybe I'm missing something?  Not sure what to do about this, but at a minimum, it seems like it would be nice to have constant, but different, temperatures on both sides of the GWE model so there is a resulting temperature gradient.  This would at least make it clear that GWE is needed for the problem.

If you decide to revise this GWE problem, then this sentence should probably be revisisted.

Reply:
@langevin-usgs I agree with your assessment and have made a number of changes to the GWE model to better illuminate the value of the viscosity package (I hope).  Changes affect the results presented in the .tex file as well as the associated discussion.  See ab1bfaf for the changes made in response to this comment.